shell/client/grain.html

```diff
@@ -72,6 +74,14 @@
   </div>
 </template>
 
+<template name="grainOwnerSuspended">
+  <div class="grain-interstitial">
+    <p>
+      You can no longer access this grain because its owner has been suspended.
```

Comment:
perhaps rephrase "because its owner's account has been suspended?"

Otherwise I have this picture of a person in midair dangling by their shoelaces.